packages/mnemonic-seed-manager/src/agent/MnemonicSeedManager.ts

```diff
@@ -139,15 +137,9 @@ export class MnemonicSeedManager implements IAgentPlugin {
     if (mnemonic) {
       const mnemonicInfo = await this.generateSeed({ mnemonic })
       if (mnemonicInfo.seed) {
-        if (args.type === 'Ed25519') {
```

Comment:
Why would you remove what makes up the interface/API even if it is the only option for now?

This change means any code calling it knowing it is working with ed25519 keys, will need to update because we will support other keys in the future as well. Just leave the argument there, but only accept ed25519 for now